backend/util/wordfactory/wordfactory.go

```diff
@@ -0,0 +1,23 @@
+package nullable
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"strings"
+)
+
+func wordGenerator() {
+	b, err := ioutil.ReadFile("wordbank.txt")
```

Comment:
Our digital ocean instance is going to be 1GB and 1Core at first, we might upgrade to a 1GB 3 core, but we should be very conscious of memory. 

I want you to look into bufio instead of ioutil